lib/controllers/actividad_controller.js

```diff
@@ -6,8 +6,8 @@ import Usuario from '../models/usuario';
 import { getCarrera } from '../helpers/api_guarani';
 
 export const index = async (req, res) => {
-  const { desde, hasta } = req.query;
-  const data = await Actividad.conAutorizaciones({ desde, hasta });
+  const { desde, hasta, inactivas } = req.query;
```

Comment:
Si seguís mi consejo, acá te conviene poner `false` por defecto, así ya en el modelo te asegurás de que viene un booleano sí o sí.

```suggestion
  const { desde, hasta, inactivas = false } = req.query;
```